server/utils/connection.js

```diff
@@ -0,0 +1,30 @@
+const { sequelize } = require("../models");
+const mongoose = require("mongoose");
+
+const startServer = async (app, port) => {
+  try {
+    // Wait for PostgreSQL to be ready
+    await sequelize.authenticate();
+    console.log("PostgreSQL connection has been established successfully.");
+
+    // Sync Sequelize models
+    await sequelize.sync();
+    console.log("PostgreSQL models synced successfully.");
```

Comment:
Would something like sequalise sync work in production as well?

Reply:
Not really, might create a lot more bugs.